npm/ipsm/ipsm.go

```diff
@@ -462,6 +462,33 @@ func (ipsMgr *IpsetManager) DeleteFromSet(setName, ip, podKey string) error {
 	return nil
 }
 
+// Clean removes all the empty sets & lists under the namespace.
```

Comment:
When do we use this function?

Reply:
Clean() function is not being currently, this is for future work items.

Reply:
I would suggest not to add function which is not used in this PR. I think this is a good practice. 
There are multiple reasons. 
1. Reviewer do not need to spend time. 
2. Add the function with the PR for your future work items. So, it will be well aligned and easy to understand PR. 
3. There are possibility that this function will not be used in the future. At that time, this function just stays for long time. 

I think you have future work item to use this function. So, add this function in the PR.